src/js/actions/layers.js

```diff
@@ -1451,14 +1452,24 @@ define(function (require, exports) {
         
         return descriptor.playObject(createObj)
             .bind(this)
-            .then(function () {
-                log.debug("Warning: calling updateDocument to add a single artboard is very slow!");
-                return this.transfer(documentActions.updateDocument);
+            .then(function (result) {
+                var payload = {
+                    documentID: document.id,
```

Comment:
I think it matches the way we do this:
```javascript
var foo = function () {
    // blah
};
```
instead of this:

```javascript
var foo = function () {
        // blah
    };
```
We make an exception when there is more than one decl though.
```javascript
var foo = 123,
    bar = function () {
        // blah
    };
```

I can be convinced otherwise, but this is the pattern I'm currently following.